spec/javascript/components/lease_ups/__snapshots__/LeaseUpsPage.test.js.snap

```diff
@@ -1279,610 +1279,7634 @@ Array [
+                >
+                  <span>
+                     
+                  </span>
+                </div>
+                <div
+                  className="rt-td td-offset-right text-right"
+                  onClick={[Function]}
+                  role="gridcell"
+                  style={
+                    Object {
+                      "flex": "100 0 auto",
+                      "maxWidth": null,
+                      "width": "100px",
+                    }
+                  }
+                >
+                  <span>
+                     
+                  </span>
+                </div>
+                <div
+                  className="rt-td td-min-wide td-status td-fixed-right"
+                  role="gridcell"
+                  style={
+                    Object {
+                      "flex": "100 0 auto",
+                      "maxWidth": null,
+                      "width": "100px",
+                    }
+                  }
+                >
+                  <span>
+                     
+                  </span>
+                </div>
+              </div>
+            </div>
+            <div
+              className="rt-tr-group"
+              role="rowgroup"
+            >
+              <div
+                className="rt-tr -padRow -odd rt-tr-status"
+                role="row"
+                style={Object {}}
+              >
+                <div
+                  className="rt-td td-min-narrow"
+                  onClick={[Function]}
+                  role="gridcell"
+                  style={
+                    Object {
+                      "flex": "100 0 auto",
+                      "maxWidth": null,
+                      "width": "100px",
+                    }
+                  }
+                >
+                  <span>
+                     
+                  </span>
+                </div>
+                <div
+                  className="rt-td text-left"
+                  role="gridcell"
+                  style={
+                    Object {
+                      "flex": "100 0 auto",
+                      "maxWidth": null,
+                      "width": "100px",
+                    }
+                  }
+                >
+                  <span>
+                     
+                  </span>
+                </div>
+                <div
+                  className="rt-td text-left"
+                  onClick={[Function]}
+                  role="gridcell"
+                  style={
+                    Object {
+                      "flex": "100 0 auto",
+                      "maxWidth": null,
+                      "width": "100px",
+                    }
+                  }
+                >
+                  <span>
+                     
+                  </span>
+                </div>
+                <div
+                  className="rt-td text-left"
+                  onClick={[Function]}
+                  role="
```

Comment:
@callaghanc it moved down here. Just bad github diffing

Reply:
ok cool